src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -94,6 +94,7 @@
 import org.apache.cassandra.locator.InetAddressAndPort;
 import org.apache.cassandra.locator.Replica;
 import org.apache.cassandra.locator.SeedProvider;
+import org.apache.cassandra.schema.CompressionParams;
```

Comment:
can we please not import schema class into this package.

this was already raised and addressed here: https://github.com/apache/cassandra/pull/2199#discussion_r1139429951